pkg/querier/querier.go

```diff
@@ -36,25 +36,39 @@ import (
 // Create a new Querier interface
 func NewV3ioQuerier(container *v3io.Container, logger logger.Logger, mint, maxt int64,
 	cfg *config.V3ioConfig, partMngr *partmgr.PartitionManager) *V3ioQuerier {
-	newQuerier := V3ioQuerier{container: container, mint: mint, maxt: maxt,
-		logger: logger.GetChild("Querier"), cfg: cfg}
+	newQuerier := V3ioQuerier{
+		container: container,
+		mint:      mint, maxt: maxt,
+		logger:            logger.GetChild("Querier"),
+		cfg:               cfg,
+		disableClientAggr: cfg.DisableClientAggr,
```

Comment:
Why did you add this attribute? `cfg` already has it.

Reply:
@imakhlin this is the default value, may be overwritten later (e.g. by promtsdb)